.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea
+*.iml
+/target
+Cargo.lock
```

Comment:
Cargo.lock is indeed pretty important and target is good too.

The others I don't mind, it seems like some IDE stuff, right?

Reply:
> Cargo.lock is indeed pretty important and target is good too.

Do you mean that `/target` and `cargo.lock` must not be in .gitignore? Or you agree?

Others are IntelliJ project files.

Reply:
I mean you are never going to commit /target and lock into the repo, right? Then they must be hidden from the git status.

Reply:
I was just voicing agreement, sorry if I confused you.

As far as I know we can add files from various IDEs here too, so I am cool with that.